.github/workflows/self-test.yml

```
@@ -0,0 +1,22 @@
+---
+name: Statick
```

I think we should rename this to `Test Action`
```suggestion
name: Test Action
```
I can't tell it apart from the other `test.yml` workflow in the github website.

And maybe rename the `test.yml` workflow to `Self Test` on line 2.